utility_scripts/compare_models.py

```diff
@@ -305,59 +313,58 @@ def prepare_plots(self, model_1_score_path, model_1_label_path, model_2_score_pa
                       model_2_label_path):
         print('Preparing plot figures.')
         figsize = self._set_figure_size(self.process_consequences)
-        constrained_layout = {'w_pad': 0.2, 'h_pad': 0.2}
         print('Preparing plots.')
-        self.fig_auc = plt.figure(figsize=figsize)
-        self.fig_auc.suptitle(
-            f'Model 1 vs Model 2 Area Under Receiver Operator Curve\n'
-            f'Model 1 scores: {model_1_score_path}\n'
-            f'Model 1 labels: {model_1_label_path}\n'
-            f'Model 2 scores: {model_2_score_path}\n'
-            f'Model 2 labels: {model_2_label_path}\n'
-        )
-        self.fig_auc.set_constrained_layout(constrained_layout)
-        self.fig_roc = plt.figure(figsize=(10, 15))
-        self.fig_roc.suptitle(
-            f'Model 1 vs Model 2 Receiver Operator Curves\n'
-            f'Model 1 scores: {model_1_score_path}\n'
-            f'Model 1 labels: {model_1_label_path}\n'
-            f'Model 2 scores: {model_2_score_path}\n'
-            f'Model 2 labels: {model_2_label_path}\n'
+        figure_supertitle = f'Model 1 scores: {model_1_score_path}\n' \
+                            f'Model 1 labels: {model_1_label_path}\n' \
+                            f'Model 2 scores: {model_2_score_path}\n' \
+                            f'Model 2 labels: {model_2_label_path}\n'
+        post_supertitle = '(M1B = Model 1 Benign, M1P = Model 1 Pathogenic, ' \
+                          'M2B = Model 2 Benign, M2P = Model 2 Pathogenic)\n'
+
+        self._prepare_plots(self.fig_auc, 'Area Under Receiver Operator Curve', figure_supertitle, figsize)
+
+        self._prepare_plots(self.fig_roc, 'Receiver Operator Curves', figure_supertitle, (10, 15))
+
+        self._prepare_plots(self.fig_afb, 'Allele Frequency bins performance comparison', figure_supertitle, (11, 11))
+
+        self._prepare_plots(
+            self.fig_score_dist_box, 'raw CAPICE score distributions', f'{figure_supertitle}{post_supertitle}', figsize
```

Comment:
Identical title to violinplots. Maybe change to something like "raw CAPICE score distributions boxplots"?